example/counter/lib/main.dart

```diff
@@ -1,132 +1,160 @@
 // ignore_for_file: avoid_print
+import 'dart:math';
+
 import 'package:dart_bindings/updater.dart';
 import 'package:flutter/material.dart';
 
 void main() {
-  Updater.loadFlutterLibrary();
-  var updater = Updater();
-  // Just to prove the bindings work at all:
-  print("active version: ${updater.activeVersion()}");
-  print("active path: ${updater.activePath()}");
+  try {
+    Updater.loadFlutterLibrary();
```

Comment:
Minor but is it necessary to create an instance of the Updater and call `loadFlutterLibrary`? It would be nice if it could be consolidated a bit.

Reply:
There are two separate versions of this, one of which takes a path, and one of which tries to load from the current process.  The caller needs to make a choice I think.  But we could also express this as an Enum on Updater() and have a default?

Reply:
This is the choice: https://github.com/shorebirdtech/shorebird/blob/main/updater/dart_bindings/lib/updater.dart#L26